openwisp_radius/api/views.py

```diff
@@ -374,9 +379,19 @@ def post(self, request, *args, **kwargs):
         self.update_user_last_login(user)
         context = {'view': self, 'request': request, 'token_login': True}
         serializer = self.serializer_class(instance=token, context=context)
-        response = {'radius_user_token': radius_token.key, 'is_active': user.is_active}
+        response = {
+            'radius_user_token': radius_token.key,
+            'is_active': user.is_active,
+            'is_verified': self._is_user_verified(user),
+        }
         response.update(serializer.data)
         status_code = 200 if user.is_active else 401
+        organization = Organization.objects.get(slug=kwargs['slug'])
```

Comment:
here too.
I guess we could move the query which retrieves the org in the _needs_identity_verification() method, passing it the keyword arguments to get.

Reply:
Done (https://github.com/openwisp/openwisp-radius/pull/237/commits/ef3686e4dbe8c5e85a5bec1769da36eaafd87ea8)